spec/fixtures/factories.rb

```diff
@@ -433,7 +433,7 @@
           "laid_date" => "2018-01-01",
           "sift_end_date" => "2018-01-05",
           "sifting_status" => "open",
-          "subject" => ["oil-and-gas"],
+          "subject" => ["business"],
```

Comment:
I don't understand why this test is failing. According to the schema definition, `oil-and-gas` is a valid value for subject: 

https://github.com/alphagov/govuk-content-schemas/blob/correct-links-schemas/formats/shared/definitions/_specialist_document.jsonnet#L1297

Reply:
`business` isn't a valid value, so the test should still fail! https://github.com/alphagov/govuk-content-schemas/blob/correct-links-schemas/dist/formats/specialist_document/publisher_v2/schema.json#L1710